packages/crew_profile_base.rb

```diff
@@ -8,6 +8,9 @@ class Crew_profile_base < Package
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  # temporary fix for crew symlink
+  FileUtils.ln_sf "../lib/crew/bin/crew", "#{CREW_PREFIX}/bin/crew" if ARGV[1].eql?('update')
```

Comment:
This won't execute either if crew_profile_base is already installed.  Take a look at line 457 in crew.

Reply:
so uprev crew_profile_base?

Reply:
I see it now, crew will read version from device.json instead of package files. Bump the version of crew profile base can make a new symlink at upgrade but not update